pkg/reconciler/client.go

```diff
@@ -74,7 +74,7 @@ func (c *client) initCustomResourceDefinitionMonitor() {
 	c.informers[CrdInformerKey] = informerFactory
 
 	// Add event handler to informer
-	c.informers[CrdInformerKey].AddEventHandler(c.crdEventHandler())
+	_, _ = c.informers[CrdInformerKey].AddEventHandler(c.crdEventHandler())
```

Comment:
can we check and do something with the errors here?

Reply:
Sure, I can include the updates from #5306 once that PR is merged.

Reply:
Updated